test/unit/raptor/RaptorDepartAfterQuery.spec.ts

```diff
@@ -29,6 +29,98 @@ describe("RaptorDepartAfterQuery", () => {
     ]);
   });
 
+  it("finds route not through via simple", () => {
```

Comment:
I'd make this test name a bit more descriptive such as: "should find a slower journey to not go via B".

Reply:
8c8b9d7 updated name